### PR TITLE
CLOUD-3128 - EAP 7.2 UBI OpenShift image

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3128
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by rgdoliveira — DISMISSED** [review]

File: cp-overrides.yaml [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-                  ref: eap72-jdk11
-          - name: jboss-eap-7-image
-            git:
-                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_720
+  repositories:
+    - name: cct_module
+      git:
+        url: https://github.com/jboss-openshift/cct_module
+        ref: master
```
Comment (bstansberry) [inline cp-overrides.yaml]: sprint-35?

Not master.

File: rel-overrides.yaml [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
-                  ref: eap72-jdk11
-          - name: jboss-eap-7-image
-            git:
-                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP72-CR5
+  repositories:
+    - name: cct_module
+      git:
+        url: https://github.com/jboss-openshift/cct_module
+        ref: master
```
Comment (bstansberry) [inline rel-overrides.yaml]: sprint-35?

Not master.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Can this be squashed when ready?

**Comment by luck3y** [comment]
Yes to squash and will fix sprints.

**Comment by luck3y** [comment]
@bstansberry Fixed and squashed.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**